Raven.Tests.Web/Controllers/Commands/AsyncCommandsController.cs

```diff
@@ -125,7 +125,7 @@ public Task<HttpResponseMessage> GetBulkInsertOperation()
         {
             using (var operation = DocumentStore.AsyncDatabaseCommands.GetBulkInsertOperation(new BulkInsertOptions(), DocumentStore.Changes()))
             {
-                operation.Write(Guid.NewGuid().ToString(), new RavenJObject(), new RavenJObject());
+                operation.WriteAsync(Guid.NewGuid().ToString(), new RavenJObject(), new RavenJObject());
```

Comment:
Either `await` or user `Write()`. You still have it.

Reply:
Ah sorry. Oversight. Will fix

Reply:
Fixed